scrapper.py

```diff
@@ -21,43 +40,160 @@ class IncorrectNumberOfArticlesError(Exception):
     """
 
 
+def _clean_text(text):
+    return re.sub(r"[\n\t ]+", " ", text).strip()
+
+
+def _get_page(link):
+    print("attempting to connect to %s", link)
+    #time.sleep(random.uniform(2.0, 4.0))
+    headers = Headers().generate()
+    print(headers)
```

Comment:
if you want to debug CI with such prints, then you need to temporarily change workflow file by adding special flags to pytest, like this: `python -m pytest --capture=no --log-cli-level=DEBUG -m "mark8 and stage_2_2_crawler_check"`

Reply:
Debugging has been removed